logging/api_reference/logging-5-7-reference.adoc

```diff
@@ -0,0 +1,7 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="logging-5-7-reference"]
+= 5.7 Logging API reference
+include::_attributes/common-attributes.adoc[]
+:context: logging-5-7-reference
+
+toc::[]
```

Comment:
I take it you are just stubbing in the assemblies now without any content (modules)?